packages/web-components/src/components/va-number-input/va-number-input.scss

```diff
@@ -10,6 +10,7 @@
 
 @import '../../mixins/form-field-error.css';
 @import '../../mixins/hint-text.css';
+@import '~@department-of-veterans-affairs/css-library/dist/elements.css';
```

Comment:
This looks good, Kerri! Did you happen to see any other opportunities in other components where we can start using this import instead of repeating the header styles?

Reply:
@powellkerry Sorry, another question before this gets merged; do we anticipate any issues importing the entire `elements.css` stylesheet into individual components like this? It has [other styles in it](https://github.com/department-of-veterans-affairs/component-library/blob/4c84048fe7cf18ea8778004fe3390a99a16add46/packages/css-library/src/stylesheets/elements.scss) besides the header styles.

I guess ultimately this stylesheet should be loaded just once globally but we have Formation still active and taking care of the general elements styles for now. But I'm wondering if we should just add this as a global import in component library now instead of later (removing it from being loaded a bunch of times individually)? 🤔

Reply:
Headings are found in alert, modal, process list, statement of truth, and summary box. A lot of those are overriding `.usa` component specific styles, so adding this change would not be helpful. Each of those components is customizing the headings in some way that is different than the global heading styles. We will probably need to discuss if they should be standardized or kept as is.

Reply:
@powellkerry Thanks for checking on that.

I think we should then just come to a consensus on where to load the `elements.css` stylesheet. It doesn't feel quite right to have to load it multiple times unless there's a component specific reason to do it.